Client/src/Components/DataCell/DataCell.js

```diff
@@ -17,6 +23,17 @@ export default function DataCell() {
         e.preventDefault();
 
         const json = e.target.jsonFile.files[0];
```

Comment:
The idea for this was not necessarily strict the upload to JSON but instead just validate that the file is an accepted type. In the future if we want to add csv or txt support then this will have to be changed

Reply:
Correct, the way we handled it will allow us to check any type we decide to add in the future because it is dependant on the extension of the file.